include/sound/soc-dpcm.h

```diff
@@ -152,10 +168,10 @@ static inline void soc_dpcm_debugfs_add(struct snd_soc_pcm_runtime *rtd)
 }
 #endif
 
-int dpcm_path_get(struct snd_soc_pcm_runtime *fe,
-	int stream, struct snd_soc_dapm_widget_list **list_);
-int dpcm_process_paths(struct snd_soc_pcm_runtime *fe,
-	int stream, struct snd_soc_dapm_widget_list **list, int new);
+int dpcm_path_get(struct snd_soc_pcm_runtime *fe, int stream,
+	struct snd_soc_dapm_widget_list **list_);
```

Comment:
@lyakh this is an unrelated change isnt it?

Reply:
@ranj063 not quite - I modified `dpcm_process_paths()` to add a new parameter and then I also moved one parameter in `dpcm_path_get()` to look similar to it. So, it isn't quite out of the blue, but it's cosmetic, if undesired, we can remove this.